revolutionpi/io_util.go

```diff
@@ -1,57 +1,64 @@
-package revolution_pi
+//go:build linux
+
+// Package revolutionpi implements the Revolution Pi.
+package revolutionpi
 
 func ioctlAddress(v int) int {
 	kbIocMagic := int('K')
 	magic := (((0) << (((0 + 8) + 8) + 14)) | ((kbIocMagic) << (0 + 8)) | ((v) << 0) | ((0) << ((0 + 8) + 8)))
 	return magic
 }
 
-const PICONTROL_NOT_CONNECTED = 0x8000
+// is the hex output received when a board is not connected.
+const piControlNotConnected = 0x8000
 
+// leaving these variables in as potential options to interface with the Revolution Pi
+//
+//nolint:unused
```

Comment:
:+1: I like that you've added a comment explaining why these are currently unused, in addition to the nolint directive. Thanks!